zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServerConf.java

```diff
@@ -60,6 +60,12 @@
      * The key in the map returned by {@link #toMap()} for the server ID.
      */
     public static final String KEY_SERVER_ID = "server_id";
+    /**
+     * The key in the map returned by {@link #toMap()} for the server socket
+     * listen backlog.
+     */
+    public static final String KEY_CLIENT_PORT_LISTEN_BACKLOG
```

Comment:
How is this class related to `ServerConfig`?
Looks like we have 2 different classes for the same purpose.

Reply:
To be honest, I really don't get it either :). I was really confused what the intention was in configuration object layering. For real installations, we seem to start from a QuorumPeerConf, but then pass this into ZooKeeperServer? Is QPC meant for quorums whereas ZKC would be relevant for just a single-server instance? Need to do more digging..

Reply:
Thanks for looking into. Not a big deal though, as long as your patch works properly, I don't want you to refactor the config layer here.